cosmo_tester/test_suites/manager_of_managers/mom_test.py

```diff
@@ -267,29 +267,29 @@ def test_regional_cluster_with_floating_ip(
     first_cluster = floating_ip_2_regional_clusters[0]
     second_cluster = floating_ip_2_regional_clusters[1]
 
-    first_cluster.deploy_and_validate()
+    first_cluster.deploy_and_validate(timeout=6000)
```

Comment:
Could it take that long? If it did is it a good result?

Reply:
Actually this could around 4000 or more so I increased it up tp 6000, this will not affect the end result, but since we are doing a lot of operations like restore from snapshot and also create resources like tenants, plugins, blueprint, and even trigger workflow execution as part of the install workflow of the cluster,